.github/workflows/linux-sampleapp-ci.yml

```diff
@@ -1,4 +1,4 @@
-name: Linux SampleApp CI
+`name: Linux SampleApps CI
```

Comment:
What's this ` is doing here?

Reply:
Accidentally pressed wrong key >-( fixed